pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/readers/forward/FixedByteChunkMVForwardIndexReader.java

```diff
@@ -195,6 +220,38 @@ private int getValueEndOffset(int rowId, ByteBuffer chunkBuffer) {
     }
   }
 
+  private long getValueEndOffsetAndRecordRanges(int chunkId, int chunkRowId, long chunkStartOffset,
```

Comment:
this method looks similar with other classes, might have a chance to extract common code to reuse

Reply:
Ack